docs/developer/slug-fields.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Slug fields
+---
+
+## Introduction
+Slug is a short label for an object. Main reason for using slugs is ability to encorporate them in URLs. Translated names can contain all utf-8 signs, which could lead to URLs which could not be written for people with different keyboard layout.
+
+Slug can contain:
+- letters
+- numbers
+- underscores
+- hyphens
+
+By convention slugs in Saleor are not translatable, so the users using different languages could use the same product URLs.
```

Comment:
Slugs are not translatable in Saleor. When using different languages users should apply the same product URLs regardless of the language.